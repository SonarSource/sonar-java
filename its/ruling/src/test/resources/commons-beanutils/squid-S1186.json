{
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils2/BeanMap.java':[
697,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils2/expression/DefaultResolver.java':[
67,
],
}
