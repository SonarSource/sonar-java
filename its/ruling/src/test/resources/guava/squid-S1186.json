{
'com.google.guava:guava:src/com/google/common/cache/AbstractCache.java':[
215,
],
'com.google.guava:guava:src/com/google/common/cache/CacheBuilder.java':[
161,
164,
167,
170,
173,
194,
],
'com.google.guava:guava:src/com/google/common/cache/LocalCache.java':[
725,
860,
883,
891,
899,
907,
915,
923,
1598,
1645,
1716,
3615,
3752,
],
'com.google.guava:guava:src/com/google/common/cache/LongAdder.java':[
61,
],
'com.google.guava:guava:src/com/google/common/collect/ComputingConcurrentHashMap.java':[
245,
285,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingMultiset.java':[
252,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingNavigableMap.java':[
276,
],
'com.google.guava:guava:src/com/google/common/collect/GenericMapMaker.java':[
56,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableListMultimap.java':[
145,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableMultimap.java':[
174,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableTable.java':[
159,
],
'com.google.guava:guava:src/com/google/common/collect/MapMaker.java':[
135,
],
'com.google.guava:guava:src/com/google/common/collect/MapMakerInternalMap.java':[
608,
730,
753,
761,
769,
777,
785,
1800,
3250,
],
'com.google.guava:guava:src/com/google/common/collect/TreeRangeMap.java':[
302,
],
'com.google.guava:guava:src/com/google/common/hash/HashingInputStream.java':[
88,
],
'com.google.guava:guava:src/com/google/common/io/ByteStreams.java':[
550,
],
'com.google.guava:guava:src/com/google/common/io/CharStreams.java':[
187,
228,
232,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/CycleDetectingLockFactory.java':[
237,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/ExecutionList.java':[
60,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/Runnables.java':[
33,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/ThreadFactoryBuilder.java':[
56,
],
}
