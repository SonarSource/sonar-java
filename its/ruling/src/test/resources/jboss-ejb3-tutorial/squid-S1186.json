{
'jboss-ejb3-tutorial:cachedentity/src/org/jboss/tutorial/cachedentity/bean/Contact.java':[
49,
],
'jboss-ejb3-tutorial:cachedentity/src/org/jboss/tutorial/cachedentity/bean/Customer.java':[
53,
],
'jboss-ejb3-tutorial:composite/src/org/jboss/tutorial/composite/bean/Customer.java':[
43,
],
'jboss-ejb3-tutorial:dependency/src/org/jboss/tutorial/dependency/bean/HasXmlMBeanDependencyBean.java':[
35,
],
'jboss-ejb3-tutorial:extended_pc/src/org/jboss/tutorial/extended/bean/Customer.java':[
41,
],
'jboss-ejb3-tutorial:extended_pc/src/org/jboss/tutorial/extended/bean/ShoppingCartBean.java':[
83,
],
'jboss-ejb3-tutorial:reference21_30/ejb21_app/src/main/java/org/jboss/tutorial/reference21_30/bean/Stateless2Bean.java':[
52,
57,
62,
67,
72,
],
'jboss-ejb3-tutorial:reference21_30/webapp/src/main/java/org/jboss/tutorial/reference21_30/servlet/EJBReferenceServlet.java':[
50,
],
'jboss-ejb3-tutorial:relationships/src/org/jboss/tutorial/relationships/bean/Customer.java':[
54,
],
'jboss-ejb3-tutorial:resource_ref/src/org/jboss/tutorial/resource_ref/bean/TestENCBean.java':[
92,
],
}
