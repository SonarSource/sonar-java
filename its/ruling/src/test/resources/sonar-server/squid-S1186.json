{
'org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/app/NullJarScanner.java':[
46,
],
}
