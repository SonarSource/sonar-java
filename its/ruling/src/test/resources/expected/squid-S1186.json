{
'project:com/sun/corba/se/impl/dynamicany/DynAnyBasicImpl.java':[
152,
],
'project:com/sun/corba/se/impl/encoding/BufferManagerReadGrow.java':[
59,
66,
85,
98,
],
'project:com/sun/corba/se/impl/encoding/BufferManagerWriteGrow.java':[
90,
],
'project:com/sun/corba/se/impl/encoding/BufferManagerWriteStream.java':[
132,
],
'project:com/sun/corba/se/impl/encoding/IDLJavaSerializationOutputStream.java':[
707,
],
'project:com/sun/corba/se/impl/interceptors/PINoOpHandlerImpl.java':[
72,
75,
78,
82,
87,
93,
96,
99,
115,
118,
121,
125,
129,
134,
138,
142,
146,
149,
153,
157,
161,
166,
171,
174,
189,
],
'project:com/sun/corba/se/impl/io/InputStreamHook.java':[
258,
262,
263,
264,
265,
],
'project:com/sun/corba/se/impl/io/OutputStreamHook.java':[
201,
202,
203,
204,
310,
],
'project:com/sun/corba/se/impl/oa/toa/TOAImpl.java':[
164,
168,
],
'project:com/sun/corba/se/impl/orb/ORBConfiguratorImpl.java':[
283,
],
'project:com/sun/corba/se/impl/orb/ORBSingleton.java':[
142,
145,
148,
584,
593,
607,
614,
618,
622,
626,
635,
644,
658,
687,
696,
710,
724,
733,
742,
747,
752,
761,
],
'project:com/sun/corba/se/impl/orb/ParserTable.java':[
456,
544,
558,
582,
609,
866,
870,
883,
887,
954,
957,
959,
961,
962,
985,
988,
990,
992,
993,
],
'project:com/sun/corba/se/impl/orbutil/ObjectWriter.java':[
169,
207,
],
'project:com/sun/corba/se/impl/orbutil/fsm/StateEngineImpl.java':[
59,
],
'project:com/sun/corba/se/impl/protocol/MinimalServantCacheLocalCRDImpl.java':[
57,
],
'project:com/sun/corba/se/spi/orbutil/fsm/StateImpl.java':[
53,
57,
],
'project:com/sun/corba/se/spi/servicecontext/UnknownServiceContext.java':[
52,
],
'project:com/sun/corba/se/spi/transport/TransportDefault.java':[
52,
],
'project:com/sun/imageio/plugins/bmp/BMPImageReader.java':[
1707,
1709,
1716,
1746,
1747,
1748,
1749,
1750,
1751,
1752,
1753,
1754,
],
'project:com/sun/imageio/plugins/bmp/BMPImageWriter.java':[
1391,
1394,
1397,
1400,
1403,
1406,
1409,
],
'project:com/sun/imageio/plugins/jpeg/JFIFMarkerSegment.java':[
1252,
1253,
1254,
1259,
1260,
1262,
1263,
1264,
],
'project:com/sun/java/swing/plaf/gtk/GTKFileChooserUI.java':[
230,
237,
],
'project:com/sun/java/swing/plaf/gtk/Metacity.java':[
687,
688,
],
'project:com/sun/java/swing/plaf/motif/MotifCheckBoxMenuItemUI.java':[
81,
99,
100,
104,
],
'project:com/sun/java/swing/plaf/motif/MotifDesktopIconUI.java':[
95,
98,
],
'project:com/sun/java/swing/plaf/motif/MotifIconFactory.java':[
360,
370,
],
'project:com/sun/java/swing/plaf/motif/MotifInternalFrameTitlePane.java':[
147,
156,
185,
186,
299,
],
'project:com/sun/java/swing/plaf/motif/MotifMenuItemUI.java':[
83,
101,
102,
106,
],
'project:com/sun/java/swing/plaf/motif/MotifMenuUI.java':[
104,
143,
144,
148,
],
'project:com/sun/java/swing/plaf/motif/MotifPopupMenuUI.java':[
106,
],
'project:com/sun/java/swing/plaf/motif/MotifRadioButtonMenuItemUI.java':[
87,
105,
106,
110,
],
'project:com/sun/java/swing/plaf/motif/MotifSliderUI.java':[
96,
99,
],
'project:com/sun/java/swing/plaf/windows/WindowsFileChooserUI.java':[
339,
350,
],
'project:com/sun/java/swing/plaf/windows/XPStyle.java':[
718,
],
'project:com/sun/jmx/interceptor/DefaultMBeanServerInterceptor.java':[
1870,
1871,
1872,
2039,
],
'project:com/sun/jmx/mbeanserver/MBeanServerDelegateImpl.java':[
115,
124,
],
'project:com/sun/jmx/mbeanserver/StandardMBeanIntrospector.java':[
104,
],
'project:com/sun/jmx/mbeanserver/StandardMBeanSupport.java':[
74,
77,
],
'project:com/sun/jmx/remote/util/EnvHelp.java':[
802,
803,
],
'project:com/sun/jmx/snmp/IPAcl/JDMAclBlock.java':[
54,
60,
],
'project:com/sun/jmx/snmp/IPAcl/JDMInformBlock.java':[
53,
59,
],
'project:com/sun/jmx/snmp/IPAcl/JDMTrapBlock.java':[
54,
60,
],
'project:com/sun/jmx/snmp/IPAcl/Parser.java':[
1244,
1247,
],
'project:com/sun/jmx/snmp/IPAcl/SimpleNode.java':[
58,
61,
],
'project:com/sun/jmx/snmp/agent/SnmpErrorHandlerAgent.java':[
62,
],
'project:com/sun/jmx/snmp/daemon/SnmpAdaptorServer.java':[
1349,
],
'project:com/sun/jmx/trace/TraceImplementation.java':[
188,
],
'project:com/sun/jmx/trace/TraceManager.java':[
250,
],
'project:com/sun/mirror/apt/AnnotationProcessors.java':[
47,
],
'project:com/sun/mirror/util/SimpleDeclarationVisitor.java':[
63,
],
'project:com/sun/mirror/util/SimpleTypeVisitor.java':[
63,
],
'project:com/sun/naming/internal/VersionHelper12.java':[
253,
],
'project:com/sun/org/apache/bcel/internal/classfile/EmptyVisitor.java':[
76,
77,
78,
79,
80,
81,
82,
83,
84,
85,
86,
87,
88,
89,
90,
91,
92,
93,
94,
95,
96,
97,
98,
99,
100,
101,
102,
103,
104,
105,
106,
107,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Parser.java':[
1214,
1232,
1370,
1375,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Whitespace.java':[
541,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/ReferenceType.java':[
360,
367,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/AdaptiveResultTreeImpl.java':[
606,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/KeyIndex.java':[
87,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/NodeSortRecordFactory.java':[
163,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SAXImpl.java':[
235,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SimpleResultTreeImpl.java':[
229,
230,
234,
546,
550,
624,
712,
716,
874,
943,
951,
996,
1000,
1009,
1013,
1017,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/DOM2SAX.java':[
358,
375,
382,
399,
408,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/DOM2TO.java':[
279,
296,
303,
320,
329,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SAX2DOM.java':[
258,
291,
313,
314,
315,
316,
317,
318,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/StAXEvent2SAX.java':[
506,
523,
530,
547,
556,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/StAXStream2SAX.java':[
462,
471,
478,
495,
504,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TrAXFilter.java':[
145,
],
'project:com/sun/org/apache/xerces/internal/dom/CoreDocumentImpl.java':[
1258,
2658,
2665,
2672,
2678,
2684,
2690,
2696,
2702,
2708,
2714,
2720,
2726,
2732,
2739,
2745,
2751,
2757,
2763,
],
'project:com/sun/org/apache/xerces/internal/dom/DOMNormalizer.java':[
1711,
1731,
1749,
1762,
1783,
1888,
1912,
1927,
1939,
2011,
2022,
2033,
2038,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLNamespaceBinder.java':[
278,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/DTDGrammar.java':[
879,
890,
909,
921,
930,
944,
954,
963,
1152,
1164,
1173,
1827,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDProcessor.java':[
428,
456,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDValidator.java':[
500,
528,
1963,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/dtd/NOTATIONDatatypeValidator.java':[
52,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/dtd/StringDatatypeValidator.java':[
57,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/XSSimpleTypeDecl.java':[
219,
221,
2864,
2867,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XMLSchemaValidator.java':[
529,
556,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/identity/XPathMatcher.java':[
159,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/opti/DefaultNode.java':[
114,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/opti/DefaultXMLDocumentHandler.java':[
76,
95,
113,
126,
147,
162,
176,
190,
211,
235,
250,
262,
279,
291,
304,
315,
326,
337,
359,
378,
414,
428,
441,
496,
511,
540,
555,
574,
593,
611,
628,
645,
658,
670,
681,
700,
715,
729,
746,
761,
773,
790,
809,
821,
832,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDComplexTypeTraverser.java':[
1152,
1155,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDHandler.java':[
4174,
4199,
4201,
4203,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/util/SimpleLocator.java':[
113,
132,
],
'project:com/sun/org/apache/xerces/internal/jaxp/JAXPValidatorComponent.java':[
581,
588,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/DOMResultAugmentor.java':[
95,
97,
99,
101,
103,
110,
114,
117,
120,
122,
184,
188,
191,
226,
228,
230,
232,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/DOMResultBuilder.java':[
208,
212,
215,
218,
220,
283,
287,
290,
331,
333,
349,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/EmptyXMLSchema.java':[
49,
55,
57,
59,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/SimpleXMLSchema.java':[
57,
63,
65,
67,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/ValidatorHandlerImpl.java':[
349,
352,
355,
393,
397,
400,
449,
451,
465,
643,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractDOMParser.java':[
1862,
1873,
2318,
2525,
2537,
],
'project:com/sun/org/apache/xerces/internal/util/SAX2XNI.java':[
190,
],
'project:com/sun/org/apache/xerces/internal/util/XMLAttributesImpl.java':[
1137,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XPointerElementHandler.java':[
604,
608,
726,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDocumentImpl.java':[
233,
2012,
2025,
2367,
2389,
2398,
2409,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMNodeProxy.java':[
2094,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/EmptyIterator.java':[
49,
51,
61,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/dom2dtm/DOM2DTM.java':[
1492,
1749,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/sax2dtm/SAX2DTM.java':[
2484,
],
'project:com/sun/org/apache/xml/internal/security/utils/ElementCheckerImpl.java':[
56,
],
'project:com/sun/org/apache/xml/internal/serialize/Printer.java':[
354,
359,
],
'project:com/sun/org/apache/xml/internal/serialize/TextSerializer.java':[
176,
181,
186,
],
'project:com/sun/org/apache/xml/internal/serializer/ToHTMLSAXHandler.java':[
95,
136,
145,
154,
164,
181,
204,
214,
257,
265,
315,
391,
399,
407,
],
'project:com/sun/org/apache/xml/internal/serializer/ToStream.java':[
2981,
2987,
],
'project:com/sun/org/apache/xml/internal/serializer/ToTextSAXHandler.java':[
127,
142,
157,
164,
171,
178,
185,
198,
211,
218,
226,
234,
241,
268,
289,
296,
303,
311,
],
'project:com/sun/org/apache/xml/internal/serializer/ToXMLSAXHandler.java':[
89,
97,
121,
128,
135,
142,
155,
162,
170,
],
'project:com/sun/org/apache/xml/internal/utils/DOMBuilder.java':[
521,
772,
789,
],
'project:com/sun/org/apache/xml/internal/utils/UnImplNode.java':[
1833,
],
'project:com/sun/org/apache/xml/internal/utils/XMLStringDefault.java':[
55,
69,
],
'project:com/sun/org/apache/xpath/internal/NodeSet.java':[
297,
],
'project:com/sun/org/apache/xpath/internal/NodeSetDTM.java':[
441,
],
'project:com/sun/org/apache/xpath/internal/domapi/XPathNamespaceImpl.java':[
112,
254,
],
'project:com/sun/org/apache/xpath/internal/functions/FuncExtFunction.java':[
249,
],
'project:com/sun/org/apache/xpath/internal/functions/FunctionMultiArgs.java':[
122,
],
'project:com/sun/org/apache/xpath/internal/objects/XNodeSet.java':[
378,
],
'project:com/sun/org/apache/xpath/internal/objects/XObject.java':[
105,
114,
135,
],
'project:java/applet/Applet.java':[
457,
489,
511,
533,
],
'project:java/awt/BorderLayout.java':[
780,
],
'project:java/awt/CardLayout.java':[
362,
],
'project:java/awt/ColorPaintContext.java':[
44,
],
'project:java/awt/Component.java':[
9030,
9033,
],
'project:java/awt/Container.java':[
2964,
],
'project:java/awt/Dialog.java':[
1060,
],
'project:java/awt/EventQueue.java':[
152,
],
'project:java/awt/FlowLayout.java':[
379,
388,
],
'project:java/awt/GridBagLayout.java':[
684,
791,
],
'project:java/awt/GridLayout.java':[
295,
302,
],
'project:java/awt/List.java':[
1299,
1302,
],
'project:java/awt/Window.java':[
1225,
2407,
3264,
4071,
],
'project:java/awt/datatransfer/StringSelection.java':[
138,
],
'project:java/awt/dnd/SerializationTester.java':[
44,
],
'project:java/awt/image/BufferedImage.java':[
1512,
1521,
1594,
],
'project:java/awt/image/IndexColorModel.java':[
1515,
],
'project:java/beans/FeatureDescriptor.java':[
425,
],
'project:java/beans/MetaData.java':[
76,
],
'project:java/beans/PropertyChangeEvent.java':[
162,
],
'project:java/beans/PropertyEditorSupport.java':[
132,
],
'project:java/beans/beancontext/BeanContextChildSupport.java':[
218,
229,
],
'project:java/beans/beancontext/BeanContextSupport.java':[
854,
869,
1152,
1161,
],
'project:java/io/ByteArrayInputStream.java':[
280,
],
'project:java/io/ByteArrayOutputStream.java':[
269,
],
'project:java/io/CharArrayWriter.java':[
271,
278,
],
'project:java/io/Console.java':[
413,
570,
],
'project:java/io/ObjectOutputStream.java':[
369,
518,
542,
],
'project:java/io/PushbackInputStream.java':[
348,
],
'project:java/io/StringWriter.java':[
225,
233,
],
'project:java/lang/Object.java':[
553,
],
'project:java/net/SocketInputStream.java':[
265,
],
'project:java/net/SocketOutputStream.java':[
176,
],
'project:java/text/DontCareFieldPosition.java':[
38,
41,
],
'project:java/util/HashMap.java':[
254,
746,
753,
],
'project:java/util/concurrent/FutureTask.java':[
131,
],
'project:java/util/concurrent/SynchronousQueue.java':[
993,
],
'project:java/util/concurrent/ThreadPoolExecutor.java':[
824,
1897,
1945,
1953,
2023,
],
'project:java/util/prefs/FileSystemPreferences.java':[
409,
],
'project:java/util/zip/InflaterInputStream.java':[
270,
],
'project:javax/imageio/metadata/IIOMetadataNode.java':[
668,
709,
],
'project:javax/management/AttributeValueExp.java':[
140,
],
'project:javax/management/ObjectName.java':[
2024,
],
'project:javax/management/StandardMBean.java':[
1118,
],
'project:javax/management/StringValueExp.java':[
89,
],
'project:javax/management/loading/MLet.java':[
793,
804,
812,
],
'project:javax/management/modelmbean/RequiredModelMBean.java':[
2838,
],
'project:javax/management/timer/Timer.java':[
212,
239,
],
'project:javax/swing/AncestorNotifier.java':[
177,
],
'project:javax/swing/BufferStrategyPaintManager.java':[
941,
944,
947,
950,
953,
],
'project:javax/swing/CellRendererPane.java':[
81,
87,
93,
],
'project:javax/swing/DefaultListCellRenderer.java':[
203,
213,
223,
231,
238,
246,
271,
279,
287,
295,
303,
311,
319,
327,
],
'project:javax/swing/GroupLayout.java':[
844,
944,
],
'project:javax/swing/JCheckBox.java':[
252,
],
'project:javax/swing/JComponent.java':[
3427,
3431,
3434,
3436,
],
'project:javax/swing/JEditorPane.java':[
2245,
2260,
],
'project:javax/swing/JFormattedTextField.java':[
1185,
],
'project:javax/swing/JInternalFrame.java':[
1841,
],
'project:javax/swing/JLayer.java':[
442,
],
'project:javax/swing/JList.java':[
3568,
],
'project:javax/swing/JMenu.java':[
214,
1111,
1620,
],
'project:javax/swing/JMenuBar.java':[
388,
396,
404,
652,
],
'project:javax/swing/JMenuItem.java':[
216,
388,
],
'project:javax/swing/JPopupMenu.java':[
1360,
],
'project:javax/swing/JRadioButton.java':[
215,
],
'project:javax/swing/JRootPane.java':[
1012,
1013,
1014,
1017,
],
'project:javax/swing/JSpinner.java':[
774,
784,
1372,
1374,
1379,
1381,
1383,
],
'project:javax/swing/JTable.java':[
7513,
7527,
7604,
7621,
7639,
7657,
9427,
],
'project:javax/swing/JTree.java':[
3265,
3273,
3281,
3290,
3300,
3310,
3320,
3331,
3342,
3770,
3772,
5119,
5177,
],
'project:javax/swing/PopupFactory.java':[
470,
636,
638,
],
'project:javax/swing/RepaintManager.java':[
1441,
1447,
1463,
1533,
1557,
],
'project:javax/swing/SpringLayout.java':[
947,
1019,
],
'project:javax/swing/SwingUtilities.java':[
1793,
1795,
1797,
1799,
1801,
1803,
],
'project:javax/swing/ToolTipManager.java':[
549,
],
'project:javax/swing/TransferHandler.java':[
1050,
1618,
1624,
1630,
1647,
1667,
1675,
],
'project:javax/swing/ViewportLayout.java':[
68,
75,
],
'project:javax/swing/border/EmptyBorder.java':[
82,
],
'project:javax/swing/colorchooser/CenterLayout.java':[
38,
39,
],
'project:javax/swing/colorchooser/DefaultSwatchChooserPanel.java':[
207,
256,
289,
],
'project:javax/swing/colorchooser/DiagramComponent.java':[
119,
122,
125,
128,
131,
],
'project:javax/swing/colorchooser/SmartGridLayout.java':[
143,
145,
],
'project:javax/swing/colorchooser/ValueFormatter.java':[
128,
],
'project:javax/swing/plaf/LayerUI.java':[
180,
212,
244,
276,
308,
340,
372,
404,
436,
445,
597,
],
'project:javax/swing/plaf/basic/BasicButtonListener.java':[
203,
207,
210,
],
'project:javax/swing/plaf/basic/BasicButtonUI.java':[
341,
347,
],
'project:javax/swing/plaf/basic/BasicColorChooserUI.java':[
247,
248,
249,
250,
],
'project:javax/swing/plaf/basic/BasicComboBoxEditor.java':[
115,
119,
],
'project:javax/swing/plaf/basic/BasicComboBoxUI.java':[
617,
662,
664,
1768,
1771,
1859,
1861,
],
'project:javax/swing/plaf/basic/BasicComboPopup.java':[
70,
71,
702,
711,
724,
726,
729,
737,
806,
866,
869,
],
'project:javax/swing/plaf/basic/BasicDesktopIconUI.java':[
243,
],
'project:javax/swing/plaf/basic/BasicDesktopPaneUI.java':[
144,
205,
208,
265,
],
'project:javax/swing/plaf/basic/BasicDirectoryModel.java':[
190,
196,
],
'project:javax/swing/plaf/basic/BasicFileChooserUI.java':[
193,
196,
380,
383,
386,
389,
496,
499,
502,
],
'project:javax/swing/plaf/basic/BasicIconFactory.java':[
121,
130,
144,
195,
203,
],
'project:javax/swing/plaf/basic/BasicInternalFrameTitlePane.java':[
187,
460,
461,
748,
784,
],
'project:javax/swing/plaf/basic/BasicInternalFrameUI.java':[
435,
1200,
1245,
1246,
1247,
1269,
1270,
1271,
1272,
1276,
1277,
1381,
1383,
1385,
1387,
1389,
1391,
1393,
],
'project:javax/swing/plaf/basic/BasicLabelUI.java':[
378,
],
'project:javax/swing/plaf/basic/BasicListUI.java':[
2439,
2639,
2642,
2645,
2779,
],
'project:javax/swing/plaf/basic/BasicMenuItemUI.java':[
865,
866,
917,
933,
],
'project:javax/swing/plaf/basic/BasicMenuUI.java':[
401,
424,
530,
546,
553,
585,
586,
628,
629,
],
'project:javax/swing/plaf/basic/BasicOptionPaneUI.java':[
989,
1112,
1221,
1224,
1227,
1230,
],
'project:javax/swing/plaf/basic/BasicPopupMenuUI.java':[
141,
267,
270,
377,
951,
952,
953,
],
'project:javax/swing/plaf/basic/BasicRadioButtonUI.java':[
198,
],
'project:javax/swing/plaf/basic/BasicRootPaneUI.java':[
74,
95,
98,
],
'project:javax/swing/plaf/basic/BasicScrollBarUI.java':[
626,
627,
],
'project:javax/swing/plaf/basic/BasicSeparatorUI.java':[
74,
78,
82,
],
'project:javax/swing/plaf/basic/BasicSliderUI.java':[
1493,
1494,
1499,
1801,
],
'project:javax/swing/plaf/basic/BasicSpinnerUI.java':[
780,
822,
],
'project:javax/swing/plaf/basic/BasicSplitPaneDivider.java':[
389,
445,
632,
1003,
1005,
],
'project:javax/swing/plaf/basic/BasicSplitPaneUI.java':[
1571,
],
'project:javax/swing/plaf/basic/BasicTabbedPaneUI.java':[
2272,
2274,
3618,
3621,
3656,
],
'project:javax/swing/plaf/basic/BasicTableHeaderUI.java':[
379,
440,
],
'project:javax/swing/plaf/basic/BasicTableUI.java':[
849,
851,
927,
1205,
1207,
1209,
],
'project:javax/swing/plaf/basic/BasicTextAreaUI.java':[
374,
389,
],
'project:javax/swing/plaf/basic/BasicTextUI.java':[
2053,
],
'project:javax/swing/plaf/basic/BasicToolBarSeparatorUI.java':[
71,
],
'project:javax/swing/plaf/basic/BasicToolBarUI.java':[
241,
245,
1154,
1197,
1198,
1199,
1200,
],
'project:javax/swing/plaf/basic/BasicTreeUI.java':[
990,
3115,
3348,
3495,
3498,
3501,
3620,
],
'project:javax/swing/plaf/basic/BasicViewportUI.java':[
71,
],
'project:javax/swing/plaf/basic/CenterLayout.java':[
38,
39,
],
'project:javax/swing/plaf/metal/MetalComboBoxUI.java':[
239,
],
'project:javax/swing/plaf/metal/MetalFileChooserUI.java':[
1172,
1234,
],
'project:javax/swing/plaf/metal/MetalIconFactory.java':[
1929,
],
'project:javax/swing/plaf/metal/MetalInternalFrameTitlePane.java':[
158,
164,
170,
210,
211,
],
'project:javax/swing/plaf/metal/MetalRootPaneUI.java':[
650,
651,
652,
655,
],
'project:javax/swing/plaf/metal/MetalSliderUI.java':[
457,
],
'project:javax/swing/plaf/metal/MetalSplitPaneDivider.java':[
113,
222,
382,
384,
],
'project:javax/swing/plaf/metal/MetalTabbedPaneUI.java':[
846,
1221,
1225,
],
'project:javax/swing/plaf/metal/MetalTitlePane.java':[
356,
875,
876,
],
'project:javax/swing/plaf/synth/SynthArrowButton.java':[
64,
],
'project:javax/swing/plaf/synth/SynthColorChooserUI.java':[
187,
],
'project:javax/swing/plaf/synth/SynthComboBoxUI.java':[
668,
692,
694,
],
'project:javax/swing/plaf/synth/SynthDesktopIconUI.java':[
201,
],
'project:javax/swing/plaf/synth/SynthDesktopPaneUI.java':[
274,
488,
],
'project:javax/swing/plaf/synth/SynthInternalFrameTitlePane.java':[
362,
363,
],
'project:javax/swing/plaf/synth/SynthInternalFrameUI.java':[
243,
],
'project:javax/swing/plaf/synth/SynthMenuBarUI.java':[
174,
],
'project:javax/swing/plaf/synth/SynthOptionPaneUI.java':[
198,
],
'project:javax/swing/plaf/synth/SynthPopupMenuUI.java':[
181,
],
'project:javax/swing/plaf/synth/SynthRootPaneUI.java':[
151,
],
'project:javax/swing/plaf/synth/SynthScrollBarUI.java':[
73,
],
'project:javax/swing/plaf/synth/SynthSpinnerUI.java':[
333,
],
'project:javax/swing/plaf/synth/SynthToolBarUI.java':[
155,
161,
257,
265,
273,
351,
353,
],
'project:javax/swing/plaf/synth/SynthViewportUI.java':[
188,
216,
],
'project:javax/swing/table/DefaultTableCellRenderer.java':[
296,
303,
310,
317,
324,
333,
359,
],
'project:javax/swing/table/JTableHeader.java':[
604,
],
'project:javax/swing/text/DefaultCaret.java':[
556,
565,
592,
],
'project:javax/swing/text/FlowView.java':[
811,
866,
],
'project:javax/swing/text/JTextComponent.java':[
4449,
4476,
4479,
4482,
],
'project:javax/swing/text/ParagraphView.java':[
847,
],
'project:javax/swing/text/TextLayoutStrategy.java':[
165,
],
'project:javax/swing/text/html/AccessibleHTML.java':[
1370,
1714,
1731,
1807,
1826,
1845,
1864,
2166,
2183,
2278,
2297,
2316,
2335,
],
'project:javax/swing/text/html/CommentView.java':[
73,
],
'project:javax/swing/text/html/HTMLDocument.java':[
2760,
2771,
3031,
3044,
],
'project:javax/swing/text/html/HTMLEditorKit.java':[
682,
1002,
1005,
1008,
1011,
1014,
1017,
1020,
1031,
1217,
1352,
1354,
1356,
],
'project:javax/swing/text/html/MinimalHTMLWriter.java':[
397,
406,
],
'project:javax/swing/text/html/StyleSheet.java':[
2993,
2994,
2995,
2996,
2997,
2998,
3000,
3001,
],
'project:javax/swing/text/html/parser/Parser.java':[
246,
261,
292,
298,
304,
],
'project:javax/swing/text/rtf/RTFReader.java':[
740,
741,
810,
811,
814,
],
'project:javax/swing/tree/DefaultTreeCellRenderer.java':[
579,
588,
595,
602,
609,
618,
641,
648,
655,
662,
669,
676,
683,
690,
],
'project:javax/swing/tree/DefaultTreeSelectionModel.java':[
1127,
],
'project:javax/swing/tree/FixedHeightLayoutCache.java':[
146,
523,
1446,
],
'project:javax/swing/tree/VariableHeightLayoutCache.java':[
685,
1444,
],
'project:org/xml/sax/ext/DefaultHandler2.java':[
70,
74,
78,
82,
86,
90,
94,
101,
106,
110,
115,
],
'project:org/xml/sax/helpers/XMLReaderAdapter.java':[
326,
337,
443,
],
}
